### PR TITLE
 [SEM-061] DESCRIPCION ARTICULO

### PR DESCRIPTION
**Pull request by Then-Crimson**
Como usuario quiero que se muestre informacion acerca de un articulo.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "@storybook/addon-links": "8.1.3",
     "@storybook/addon-onboarding": "8.1.3",
     "@storybook/blocks": "8.1.3",
-    "@storybook/react": "8.1.3",
+    "@storybook/react": "^8.1.3",
```
Comment (Ditmar) [inline package.json:49]: please don't use the carret into the version please removed

File: src/ui/components/atoms/article-description/styles.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+.article-text{
+    font-family: Lato, sans-serif;
+    font-size: $ads-size-main;
+    letter-spacing: 0.125rem;
+    line-height: 2.05rem;
+    margin-bottom: $ads-letter-spacing;
+
+    /* word-spacing: $ads-letter-spacing; */
```
Comment (Ditmar) [inline src/ui/components/atoms/article-description/styles.scss:12]: remove this comment

**Review by Ditmar — CHANGES_REQUESTED** [review]

**Comment by DanX3N0** [comment]
PR:
image
figma:
image

not according to figma, check the weight of typography

**Comment by DanX3N0** [comment]
FIGMA:
image

PR:
image
image
image
image
image

it's according to figma

File: src/ui/components/atoms/article-description/ArticleDescription.test.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
```
Comment (Ditmar) [inline src/ui/components/atoms/article-description/ArticleDescription.test.jsx:1]: please change the extension of the file to tsx

**Review by Ditmar — COMMENTED** [review]
Nice work

**Review by Ditmar — APPROVED** [review]

**Pull request merged**